crates/misc/dotnet/docs/articles/intro.md

```diff
@@ -0,0 +1,214 @@
+# Introduction to Wasmtime for .NET
+
+[Wasmtime](https://github.com/bytecodealliance/wasmtime) is a standalone runtime capable of executing [WebAssembly](https://webassembly.org/) outside of a web browser.
+
+Wasmtime for .NET is a .NET API for Wasmtime.  It enables .NET developers to easily instantiate and execute WebAssembly modules.
+
+For this tutorial, we will create a WebAssembly module from a program written in Rust and use that WebAssembly module from a .NET Core 3.0 application.
+
+# Creating a simple WebAssembly module
+
+One of the reasons why WebAssembly is so exciting is that [many languages are able to target WebAssembly](https://github.com/appcypher/awesome-wasm-langs).  This means, for example, a plugin model based on WebAssembly could enable developers to write sandboxed, cross-platform plugins in any number of languages.
+
+Here I've decided to use [Rust](https://www.rust-lang.org/) for the implementation of the WebAssembly module.  Rust is a modern systems programming language that can easily target WebAssembly.
+
+If you wish to skip creating the WebAssembly module, download the [prebuilt WebAssembly module](https://raw.githubusercontent.com/bytecodealliance/wasmtime/master/crates/misc/dotnet/docs/wasm/intro/hello.wasm) from this tutorial, copy it to your .NET project directory, and continue from the _[Using the WebAssembly module from .NET](#using-the-webassembly-module-from-net)_ section.
+
+## Installing a Rust toolchain
+
+To get started with Rust, install [rustup](https://rustup.rs/), the manager for Rust toolchains.
+
+This will install both a `rustup` command and a `cargo` command (for the active Rust toolchain) to your PATH.
+
+## Installing the WebAssembly target
+
+To target WebAssembly with the active Rust toolchain, install the WebAssembly [target triple](https://forge.rust-lang.org/release/platform-support.html):
+
+```text
+rustup target add wasm32-unknown-unknown
+```
+
+## Creating the Rust project
+
+Create a new Rust library project named `hello`:
+
+```text
+cargo new --lib hello
+cd hello

```

Comment:
This may want to prefer to link to a page like https://bytecodealliance.github.io/wasmtime/wasm-rust.html to avoid duplicating how to set up a wasm binary in a few places perhaps?

Reply:
That's a good idea.  I'll see how I can work that into this introduction article.

Reply:
I'm going to leave this as-is for now.  I'm writing up a new article that will replace this one soon and I will make reference to this page as well as `cargo wasi`.